src/pandas_profiling/model/summary_algorithms.py

```diff
@@ -36,11 +36,13 @@ def histogram_compute(
     stats = {}
     bins = config.plot.histogram.bins
     bins_arg = "auto" if bins == 0 else min(bins, n_unique)
-    stats[name] = np.histogram(finite_values, bins=bins_arg, weights=weights)
+    bins = np.histogram_bin_edges(finite_values, bins=bins_arg)
+    stats[name] = np.histogram(finite_values, bins=bins, weights=weights)

```

Comment:
interesting solution, but still seems to behave a bit weirdly with big numbers
```python
> import numpy as np
> arr = np.array([716277643516076032 + i for i in range(100)])
> bins = np.histogram_bin_edges(arr, bins=5)
> np.histogram(arr, bins=bins)
(array([ 0,  0, 65,  0, 35]),
 array([7.16277644e+17, 7.16277644e+17, 7.16277644e+17, 7.16277644e+17,
        7.16277644e+17, 7.16277644e+17]))
```

Reply:
True, it is still not evenly distributed like it should be for smaller numbers. What do you propose here? Leaving `np.histogram_bin_edges` raises an error for larger numbers. Is it better to raise error than have weird behavior?

Reply:
Thinking from a user's perspective, for me is better to have an error being raised than an incorrect plot. If I know that there was a problem with the large integers I can preprocess that column and run again, but an incorrect result may lead me to an incorrect interpretation of my data distribution.

Reply:
Yeah, that is what I was thinking as well. I think I should make the changes so that it leads to raising an error rather than making an incorrect plot, right?